vendor/assets/stylesheets/dvl/core/typography.scss

```diff
@@ -6,18 +6,19 @@
    http://modularscale.com/scale/?px1=12&px2=20&ra1=1.333&ra2=0
 \* ================================================================== */
 
-html { font-size: 75%; } // 12px
-@media only screen and (min-width: 320px) { html { font-size: $maxFontSize * 0.8125; } } // 13px
```

Comment:
The bugs you mentioned in that comment have been fixed.

On Wed, Jul 8, 2015 at 11:15 AM, Adam Becker <notifications@github.com>
wrote:

> In vendor/assets/stylesheets/dvl/core/typography.scss
> <https://github.com/dobtco/dvl-core/pull/105#discussion_r34180043>:
>
> > @@ -6,18 +6,19 @@
> >     http://modularscale.com/scale/?px1=12&px2=20&ra1=1.333&ra2=0
> >  \* ================================================================== */
> >
> > -html { font-size: 75%; } // 12px
> > -@media only screen and (min-width: 320px) { html { font-size: $maxFontSize * 0.8125; } } // 13px
>
> I'm pretty sure this is what's causing bad layouts in small viewports: dobtco/screendoor-v2#2153
> (comment)
> <https://github.com/dobtco/screendoor-v2/pull/2153#issuecomment-117712592>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dobtco/dvl-core/pull/105/files#r34180043>.
>